src/SpectatorView.Unity/Assets/SpatialAlignment/README.md

```diff
@@ -1,60 +1,67 @@
+
+3. Depending on the experience, the SpatialCoordinateSystemManager is directed to create a LocalizationSession for a specific SpatialLocalizer. For experiences with the Spectator role chosen on the SpectatorView component, a SpatialLocalizationInitializer that controls localization is chosen from one of two locations. The experience can choose a preferred list of SpatialLocalizationInitializers on the SpectatorViewSettings prefab. In Unity, the Spectator View / Edit Settings menu option will generate this prefab if it does not exist. On the SpatialLocalizationInitializationSettings component, the PrioritizedInitializers list can specify one or more SpatialLocalizationInitializer components. At runtime, the first SpatialLocalizationInitializer that is supported by both devices will be chosen and used to start spatial alignment. If no app-specific SpatialLocalizationInitializer has been specified on the SpectatorViewSettings prefab, a default set specified by the SpectatorView prefab will be consulted in the same manner. For DSLR filming, the compositor window in the Unity editor tells both the user HoloLens and HoloLens mounted to the DSLR camera when to localize. LocalizationSession logic varies for different spatial alignment strategies. A more in-depth look at each alignment strategy can be found below.
+
+4. Once the LocalizationSession has completed, a SpatialCoordinate will have been located. The position of this SpatialCoordinate in the local application space is then cached in the SpatialCoordinateSystemParticipant. The SpatialCoordinateSystemParticipant then relays this information to its associated peer device. This allows both devices to know the location of the SpatialCoordinate in each others' local application spaces.
+
+5. Once the SpatialCoordinate's location and orientation are known for both devices, the spectator device's camera is updated so that the observed content has a physical origin that's at the same location as the user device's local application origin. For mobile devices, this is achieved through the SpatialCoordinateTransformer class that applies an additional transform to the mobile device's tracking camera. For DSLR filming, the editor combines reported head pose information with this SpatialCoordinate information to update its camera.
+
+> Note: Spatial alignment relies on updating the Unity camera transform on spectator devices for aligning experiences in the physical world. Its possible to move application content instead of transforming Unity cameras to align scene content in the physical world. However, transforming application content can make for a more difficult time synchronizing physics state information across devices, so its suggested to update the camera transform.
+
+## Spatial Alignment Strategies
+
+### Azure Spatial Anchors
```

Comment:
Will this section be in a follow up review?

Reply:
Yes, this review doesn't really add new documentation.